docs/releases/1.8.txt

```diff
@@ -1131,6 +1131,11 @@ Miscellaneous
   has been removed by a migration and replaced by a property. That means it's
   not possible to query or filter a ``ContentType`` by this field any longer.
 
+* :djadmin:`migrate` now accepts the :djadminopt:`--fake-initial` option to
+  allow faking initial migrations. In 1.7 initial migrations were automatically
+  faked if all tables created in an initial migration already existed without
```

Comment:
(I'd also remove it because mentioning it here makes it sound like we've changed that aspect, which we haven't. It still doesn't check for matching schema.)